declarations_site/catalog/static/css/decls.css

```diff
@@ -242,3 +246,15 @@ td.l-weiss-form__item {
     padding: 0;
     position: absolute !important;
 }
+
+/* @group Currency align
+  --------------------------------------------------------------------------- */
+
+.currency-empty {
+    display: block;
+    text-align: center;
+    font-style: normal;
+    font-family: "Times New Roman", Times, "FreeSerif", "Nimbus Roman No9 L", serif;
```

Comment:
Could you please change font color to be grey, not black. It'll help to reduce visual noise a bit